rest/route/distro.go

```diff
@@ -156,7 +156,7 @@ func (h *distroIDPutHandler) Parse(ctx context.Context, r *http.Request) error {
 // (b) creates a new resource based on the Request-URI and JSON payload
 func (h *distroIDPutHandler) Run(ctx context.Context) gimlet.Responder {
 	user := MustHaveUser(ctx)
-	original, err := distro.FindByID(h.distroID)
+	original, err := distro.FindOneId(h.distroID)
 	if err != nil {
```

Comment:
|| original == nil

Reply:
There is already casing for the original being nil on line 196

Reply:
I don't think this route wants to treat a nil distro as an error